src/app/shell/personal-cabinet/provider/provider-drafts/provider-drafts.component.spec.ts

```
@@ -0,0 +1,193 @@
+      }
+      // @ts-ignore
+      if (selector === RegistrationState.role) {
+        return of(Role.provider);
+      }
+    });
+
+    fixture = TestBed.createComponent(ProviderDraftsComponent);
+    component = fixture.componentInstance;
+    component.provider = { id: '123' } as Provider;
+    component.ngOnInit(); // to set role
+  });
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)